round_test.go

```diff
@@ -161,6 +161,18 @@ func testRounding(t *testing.T, a, b string, prec int, method RoundingMode) {
 	if x.Cmp(y) != 0 {
 		t.Errorf("test Round(%v, %v, %v) == %s. Got %v", a, prec, method, y, x)
 	}
+
+	if prec == 0 {
+		x, _ := new(big.Rat).SetString(a)
```

Comment:
hm, maybe should add the `ok` there too, in the case someone copypastes it later...